core/arch/riscv64/emit_utils.c

```diff
@@ -661,6 +663,20 @@ append_restore_xflags(dcontext_t *dcontext, instrlist_t *ilist, bool absolute)
             INSTR_CREATE_csrrw(dcontext, opnd_create_reg(DR_REG_ZERO),
                                opnd_create_reg(DR_REG_A0),
                                opnd_create_immed_int(VCSR, OPSZ_12b)));
+
+        /*
+         * The code below is contributed and copyrighted by FORTH
+         * Copyright (c) 2024 Foundation of Research and Technology, Hellas.
```

Comment:
We do not use partial file copyright delimiters: please add just this line to the top of the file by the other copyright notices.

Reply:
Done